packages/ui-components/src/tailwind.css

```diff
@@ -105,8 +108,9 @@
     --primary: 0 0% 100%;
     --primary-100: 204 38% 75%;
     --primary-200: 239 65% 61%;
-    --primary-300: 250 75% 12%;
+    --primary-300: 265 79% 71%;
```

Comment:
this color was too dark, so you could not read the text in darkmode (as seen in chromatic diffs)